lms/djangoapps/courseware/views/views.py

```diff
@@ -753,10 +753,17 @@ def _progress(request, course_key, student_id):
         'passed': is_course_passed(course, grade_summary),
         'show_generate_cert_btn': show_generate_cert_btn,
         'credit_course_requirements': _credit_course_requirements(course_key, student),
-        'missing_required_verification': missing_required_verification
+        'missing_required_verification': missing_required_verification,
+        'certificate_invalidated': False,
     }
 
     if show_generate_cert_btn:
+        # If current certificate is invalidated by instructor
+        # then don't show the generate button.
```

Comment:
This comment is confusing. Can you update the comment to `then show the certificate invalidated message`